front/app/components/UI/Button/index.tsx

```diff
@@ -1,31 +1,66 @@
-import React, { memo } from 'react';
+import React from 'react';
 import { isNilOrError } from 'utils/helperUtils';
 import useLocale from 'hooks/useLocale';
 import {
   Button,
   ButtonProps,
   ButtonContainerProps as ComponentLibraryButtonContainerProps,
-  ButtonStyles as ComponentLibraryButtonStyles,
+  ButtonStyles,
 } from '@citizenlab/cl2-component-library';
 
-interface Props extends Omit<ButtonProps, 'locale'> {}
+import Link from 'utils/cl-router/Link';
+interface Props extends ButtonProps {
+  linkTo?: string;
+  openLinkInNewTab?: boolean;
+}
 
 interface ButtonContainerProps extends ComponentLibraryButtonContainerProps {
   'data-testid'?: string;
 }
 
-type ButtonStyles = ComponentLibraryButtonStyles;
-
-const ButtonWrapper = memo<Props>((props) => {
+const ButtonWrapper = ({ linkTo, openLinkInNewTab, ...rest }: Props) => {
   const locale = useLocale();
+  const isExternalLink =
+    linkTo && (linkTo.startsWith('http') || linkTo.startsWith('www'));
+
+  const link = linkTo
+    ? isExternalLink
+      ? ({
+          children,
+          ...rest
+        }: ButtonProps & React.HTMLAttributes<HTMLAnchorElement>) => (
+          <a
+            href={linkTo}
+            target={openLinkInNewTab ? '_blank' : undefined}
+            rel="noreferrer"
+            {...rest}
+          >
+            {children}
+          </a>
+        )
+      : ({
+          children,
+          ...rest
+        }: Omit<ButtonProps, 'as' | 'size'> &
+          React.HTMLAttributes<HTMLAnchorElement>) => (
+          <Link
+            to={linkTo}
+            target={openLinkInNewTab ? '_blank' : undefined}
```

Comment:
I think this and the next line could be removed from the internal link block here, as I don't think the referrer matters, and internal router links will always open in the same page. We can double check though

Reply:
Not necessarily - sometimes you want them to open on a new tab (like the project preview, for example). Doesn't hurt to leave it, I think.

Reply:
Good point, it's fine that way then